utils/base_class.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+import requests
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.select import Select
+
+@pytest.mark.usefixtures("setup")
```

Comment:
oops I didn't know you already have a base class
the idea of base classes is that they are inherited, and then anything they have (attributes and methods) automatically is transfered to their child classes, too. So all the Page classes should inherit the BaseClass. And BaseClass should contain the code that can be reused in all pages

Reply:
All pages based on BaseClass now